src/mixins/allowable-actions-mixin.ts

```diff
@@ -31,8 +31,9 @@ export default class AllowableActionsMixin extends Vue {
       }
 
       case AllowableActions.ADMINISTRATIVE_DISSOLUTION: {
-        // FUTURE: check dissolution type
-        return this.isAllowedFiling(FilingTypes.DISSOLUTION)
+        // NB: specific entities are targeted via LaunchDarkly
+        const ff = !!GetFeatureFlag('supported-dissolution-entities')?.includes(this.getLegalType)
+        return (ff && this.isAllowedFiling(FilingTypes.DISSOLUTION, FilingSubTypes.DISSOLUTION_ADMINISTRATIVE))
```

Comment:
Now is the future -- I added the subtype check. I also added the FF check that should have been there.

Reply:
Nice!